packages/client/utils/getTokenFromSSO.ts

```diff
@@ -16,6 +17,19 @@ const getTokenFromSSO = (url: string) => {
       if (!token && !error) return
       if (event.origin !== window.location.origin) return
 
+      const params = new URLSearchParams(popup.location.search)
+      const userId = params.get('userId')
+      const isNewUser = params.get('isNewUser') === 'true'
+      const isPatient0 = params.get('isPatient0') === 'true'
+      if (isNewUser && !error) {
+        ReactGA.event('sign_up', {
+          userId,
+          user_properties: {
+            is_patient_0: isPatient0
+          }
+        })
+      }
```

Comment:
I know this is kinda weird to put it here... but this appears the only place we can access the SAML redirect URL in the popup window. Maybe we can rename this method? Or make it return more data than `token` and handle the GA4 stuff in locations where this method is called, i.e., `EmailPasswordAuthForm` and `TeamInvitationSSO`?

Reply:
-1 I agree. We're not deciding here that it was a successful login, but on the calling side https://github.com/ParabolInc/parabol/blob/05e7f9336c9a94b37d8e2589db51ba773a95c90a/packages/client/components/EmailPasswordAuthForm.tsx#L180 so that's a better place to send the event